index.html

```diff
@@ -260,5 +251,38 @@ <h2 class="heading-font">About me</h2>
         </section>
     </section>
 
+    <footer class="d-flex">
+        <h2 class="inter-font" id="footer-title">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </h2>
+        <form class="d-flex" action="https://formspree.io/f/xwkawbaz" method="POST" name="contact" id="contact-form">
+            <input class="inter-font input-contact" type="text" name="user_name" placeholder="Full name"  maxlength="30" required>
+            <input class="inter-font input-contact" type="email" name="user_email" placeholder="Email address" required>
+            <textarea class="inter-font" id="area-contact" name="user_text" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] I do see that the `textarea` has **`required`** but I still can send an empty message, please fix it.